packages/frontend/src/utils/token.ts

```diff
@@ -0,0 +1,42 @@
+export const BRIDGED_CONSTANT = 'Bridged';
+
+export const formatToken = (token: Wallet.Token) => {
```

Comment:
There's a better way to do this using an object / Map technique, as opposed to `if/return` statements.

Reply:
I thought about the object map but didn't think it read well, don't think the if statements in this scenario is a bad idea since we have two checks. 

Not sure I like your implementation to be honest, not as clear as the if statements and you need to add comments so people know what's going on.

I'm not too precious about this though so we can go your way if you'd like.

Reply:
I agree with what @DominicGBauer mentioned here: the if-else implementation here seems more direct and digestible.

Reply:
Using if-else for what is essentially a static map of key -> metadata, to me that's never the correct pattern, even if its a tiny list. The concept we are creating here is about "potentially" overriding contract data based on its key, that should mean we always reach for a Map pattern (if key in Map, use metadata, else do nothing).